src/IndexRegistry.sol

```diff
@@ -119,10 +134,10 @@ contract IndexRegistry is IndexRegistryStorage {
      * @param index the latest index of that operator in the list of operators registered for this quorum
      */ 
     function _updateOperatorIdToIndexHistory(bytes32 operatorId, uint8 quorumNumber, uint32 index) internal {
-        OperatorUpdate memory latestOperatorUpdate;
-        latestOperatorUpdate.operatorId = operatorId;
-        latestOperatorUpdate.fromBlockNumber = uint32(block.number);
-        _indexToOperatorIdHistory[quorumNumber][index].push(latestOperatorUpdate);
+        _indexToOperatorIdHistory[quorumNumber][index].push(OperatorUpdate({
+            operatorId: operatorId,
+            fromBlockNumber: uint32(block.number)
+        }));
```

Comment:
it's weird that elsewhere we use 24 bytes (as in the BLSPubkeyRegistry) for the equivalent of operatorId, and here we use 32 bytes. Not sure why this line in particular made me notice this, but it's just...a bit odd?
should operatorIds perhaps be shorter, so we can do the same storage packing everywhere?